src/NexusMods.App.UI/Overlays/Download/Cancel/CancelDownloadOverlayView.axaml.cs

```diff
@@ -9,11 +9,17 @@ public partial class CancelDownloadOverlayView : ReactiveUserControl<ICancelDown
     public CancelDownloadOverlayView()
     {
         InitializeComponent();
+
         this.WhenActivated(_ =>
         {
-            OkCancelView.Description =
+
+            OkCancelView.Description = ViewModel?.DownloadTasks.Count() == 1 ?
+                string.Format(Language.CancelDownloadOverlayView_Description_download_will_be_cancelled,
+                    ViewModel?.DownloadTasks.First().Name + " " +
+                    Language.CancelDownloadOverlayView_Description__Download) :
                 string.Format(Language.CancelDownloadOverlayView_Description_download_will_be_cancelled,
-                    ViewModel?.DownloadTask.Name);
+                    ViewModel?.DownloadTasks.Count() + " " +
+                    Language.CancelDownloadOverlayView_Description_downloads );
```

Comment:
This isn't reactive? You should still use a `PopulateFromViewModel` method for consistency.

Reply:
The number of tasks to cancel doesn't change during the life of the dialog. I updated the code to make it more standard.